### ISSUE TITLE
[SOLVED] Meteor 1.4 Viewmodel events don't work

### ISSUE THREAD

**Issue by scott-cornwell** [ManuelDeLeon/viewmodel#274]
Events don't seem to work with Meteor 1.4

**Comment by ManuelDeLeon** [comment]
Make a repro. Events work just fine.

**Comment by fvpDev** [comment]
Don't forget the change in vm where `events` is not a definable property anymore because it's better to use bindings (especially since there is the `repeat` function).
See here: https://viewmodel.org/docs/bindings#events

Something I would forget sometimes, so I hope this helps.

**Comment by ManuelDeLeon** [comment]
@fvpDev `repeat` is React only :)

**Comment by fvpDev** [comment]
Oh, whoops. Now I bet ur really annoyed lol sorry about that. Thanks for the clarification.

**Comment by scott-cornwell** [comment]
@ManuelDeLeon Sorry I just solved my issue. I was starting a new Meteor 1.4 project and doing `meteor add manuel:viewmodel` gave me viewmodel version 1.9.12 for some reason! I checked that the latest is 6 and tried to update but no luck. Seems to be a Meteor bug. Had to do `meteor add manuel:viewmodel@6.0.0` and it updated and now events are working!

I will say there is still some confusion on my part. I guess the version of viewmodel I was using with Meteor 1.3 was 4.1.9. It seems as though {{b "value: message"}} syntax no longer works. I was able to get it working with b="value: message" syntax. Is there somewhere that these changes are mentioned? I still see a lot of examples on your site using the presumably old method, and also the events object on viewmodel (which seems to still work!?). Actually I have that working with Meteor 1.4 but not the event binding method. 

Anyhow, thanks for all your work, viewmodel is great! I think I have it working now even though there is some guessing and confusion still on my part.
